### ISSUE TITLE
CoveragePkg:   Issue with GetNextPoint(Mode => MIN)

### ISSUE THREAD

**Issue by JimLewis** [OSVVM/OSVVM#48]
There is a potential bug with:    GetNextPoint(Mode => MIN);

For example, ModelSim reports :
```
# -- Compiling architecture behav of tb_ldpc_encoder
# ** Error (suppressible): testbench/tb_ldpc_encoder_osvvm.vhd(240): (vcom-1380) Identifier "MIN" is not directly visible.
#
#    Potentially visible declarations are:
#         std.STANDARD.min (physical unit)
#         osvvm.CoveragePkg.MIN at testbench/osvvm_2020_05/CoveragePkg.vhd(161) (enumeration literal)
```

Fortunately, this can be easily fixed with calling “MIN” explicitly from Coverage package:

GetNextPoint(Mode => osvvm.CoveragePkg.MIN);

Maybe it is good idea to record this issue somewhere:

Thanks you and best regards,

Miroslav

**Comment by JimLewis** [comment]
Hi Miroslav,
Thanks.

I am aware of it.   I saw it in my testing.   It is a conundrum as MINIMUM caused a problem on at least one simulator.   I probably have to give up on the name MIN as that may also cause problems for MIN as a time unit.  Although it should not be an issue.

The current plan is to change the parameter value back to MINIMUM and test it with other simulators.   I suspect the issue I encountered with MINIMUM was a simulator issue and not a language issue - it can be dealt with in the same manner you highlighted within that simulator only then.

Best Regards,
Jim
